lib/bundled_gems.rb

```diff
@@ -105,11 +105,11 @@ def self.warning?(name, specs: nil)
       # and `require "syslog"` to `require "#{ARCHDIR}/syslog.so"`.
       feature.delete_prefix!(ARCHDIR)
       feature.delete_prefix!(LIBDIR)
-      segments = feature.split("/")
+      segments = feature.split("/", 3)
```

Comment:
This branch needs to have three segments at most.

Reply:
Ah, because only two shifts. 🤦

Reply:
Yes two shifts and then just one needed for the `segments.any?` return value. I'll add a comment 👍🏽